module5/NewsFeedStatistic.py

```diff
@@ -0,0 +1,55 @@
+import csv
+import re
+from typing import Dict
+
+
+class NewsFeedStatistic:
+
+    def __init__(self, newsfeed_file_name):
+        self.newsfeed_file_name = newsfeed_file_name
+        self.text = self.__read_newsfeed()
+        self.letters = self.__get_letters()
+
+    def write_word_statistic(self, csv_file_name) -> None:
+        with open(csv_file_name, 'w') as csvfile:
+            writer = csv.writer(csvfile, delimiter='-')
+            for word, count in self.__get_world_statistic().items():
```

Comment:
I think items is working not properly on arabic keys... It creates corteges with values as part of cortege with index = 0 and key as part of cortege with index = 1. That is why csv file will return bad data... I think we need to handle this situation somehow